packages/shorebird_cli/lib/src/commands/patch/patch_ios_command.dart

```diff
@@ -378,8 +379,9 @@ Please re-run the release command for this version or create a new release.''');
             '🟢 Track: ${lightCyan.wrap('Production')}',
           if (percentLinked != null)
             '''🔗 Running ${lightCyan.wrap('${percentLinked.toStringAsFixed(1)}%')} on CPU''',
-          if (results['debug-linker'] == true)
-            '''🔍 Debug Info: ${lightCyan.wrap(_debugInfoOutpath)}''',
+          if (results['debug-linker'] == true &&
```

Comment:
As discussed, we'll need to guard against passing this flag since older versions of aot_tools don't support `--dump-link-info`.

Reply:
Just added some code to verify it. Can you take a look please?